openlibrary/plugins/openlibrary/api.py

```diff
@@ -324,6 +324,8 @@ def GET(self, _id):
             else models.Edition.from_isbn(_id)
             
         )
+        if not edition:
```

Comment:
This is an insufficient error check as this throws the statement "Invalid ISBN 13" for valid ISBN values. They just happen not to be eligible for sponsorship. This is not differentiating between valid and invalid ISBN values.

Reply:
Can you please provide a valid yet _falsey_ ISBN value?

Reply:
See - https://github.com/internetarchive/openlibrary/pull/3168#issuecomment-600141196 (sorry for the duplicate. Responded on my phone and apparently wasn't replying to this comment directly.

Reply:
I have to agree with @brierjon that this is a poorly worded error message since it's just reporting a lookup failure, which could happen with an invalid ISBN, but also for lots of other reasons.